frontend/src/pages/profile/Profile.tsx

```
@@ -350,19 +336,8 @@ export default function Profile() {
-  // Helper function to get username by ID (Simplified)
-  async function getUsername(userId: string): Promise<string> {
-    try {
-      const userResponse = await axios.get(`${apiURL}/user/${userId}`);
-      return userResponse.data.username || "Unknown User"; // Provide a fallback
-    } catch (error) {
-      console.error("Error fetching username:", error);
-      return "Unknown User"; // Fallback in case of error
-    }
-  }
+    (user && user.uid === id) ||
+    (user && user.uid && user?.followers?.includes(user.uid));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)